content/en/docs/Examples/services.md

```diff
@@ -716,6 +850,25 @@ service:
       icon: https://github.com/nextcloud/server/raw/master/core/img/favicon.png
 ```
 
+## OliveTin/OliveTin
+Source: https://github.com/OliveTin/OliveTin
+```yaml
+service:
+  OliveTin:
+    options:
+      semantic_versioning: false
```

Comment:
`OliveTin/OliveTin` appears to follow semantic versioning

Reply:
When I add the semantic versioning it collects the version from github without the 0, and when the semantic versioning is off it shows the correct version 
![image](https://github.com/user-attachments/assets/67ed6798-6f5b-40b5-934a-1dde8a4d42e6)
![image](https://github.com/user-attachments/assets/d3311fbe-7d3d-44a0-8bdb-751d17c09054)

That's my reason why I removed semantic versioning everywhere, not sure why it does that, maybe it's my error, I'm not doing it properly? Should I use regEx?

Reply:
Ah yeah didn't notice the leading 0's sorry. Best to keep this one as non-semantic